Makefile

```diff
@@ -146,9 +120,18 @@ credentials/backup-secret.json:
 	@mkdir -p credentials
 	@echo '$(BACKUP_SECRET)' > credentials/backup-secret.json
 
-ostree-backup: credentials/backup-secret.json ## Backup model VM into ostree container		make ostree-backup SEED_IMAGE=quay.io/whatever/ostmagic:seed
-	scp $(SSH_FLAGS) ostree-backup.sh credentials/backup-secret.json core@$(MODEL_VM_NAME):/tmp
-	ssh $(SSH_FLAGS) core@$(MODEL_VM_NAME) sudo /tmp/ostree-backup.sh $(SEED_IMAGE)
+.PHONY: ibu-imager
+ibu-imager: credentials/backup-secret.json ## Create seed image using ibu-imager		make ibu-imager SEED_IMAGE=quay.io/whatever/ostmagic:seed
+	scp $(SSH_FLAGS) credentials/backup-secret.json core@$(MODEL_VM_NAME):/tmp
+	ssh $(SSH_FLAGS) core@$(MODEL_VM_NAME) sudo podman run --privileged --rm --pid=host --net=host \
+		-v /var:/var \
+		-v /var/run:/var/run \
+		-v /etc:/etc \
+		-v /run/systemd/journal/socket:/run/systemd/journal/socket \
+		-v /tmp/backup-secret.json:/tmp/backup-secret.json \
+		--entrypoint /ibu-imager \
```

Comment:
Isn't this already covered by https://github.com/openshift-kni/lifecycle-agent/blob/a87fc174d743132c60d0e6b437f626c879f8bdeb/ibu-imager/Dockerfile#L39 ?

Reply:
Now there's a single image for lifecycle agent and ibu-imager, so for using ibu-imager we need to specify entrypoint:
https://github.com/openshift-kni/lifecycle-agent/blob/a87fc174d743132c60d0e6b437f626c879f8bdeb/Dockerfile#L54-L60

I guess the Dockerfile that you're refering to is legacy, or for development purposes, maybe @leo8a can clarify?

Reply:
ack